app/server/lib/OIDCConfig.ts

```diff
@@ -39,7 +39,8 @@
  *        A comma-separated list of protections to enable. Supported values are "PKCE", "STATE", "NONCE"
  *        (or you may set it to "UNPROTECTED" alone, to disable any protections if you *really* know what you do!).
  *        Defaults to "PKCE,STATE", which is the recommended settings.
- *        It's highly recommended that you enable STATE, and at least either PKCE or NONCE.
+ *        It's highly recommended that you enable STATE, and at least either PKCE or NONCE
+ *        depending on what requires/supports your OIDC provider.
```

Comment:
Just for a slightly clearer English here, I suggest "It's highly recommended that you enable STATE, and at least one of PKCE or NONCE, depending on what your OIDC provider requires/supports."

Reply:
Thanks, done!